tools/syz-trace2syz/parser/parser.go

```diff
@@ -0,0 +1,99 @@
+package parser
+
+import (
+	"bufio"
+	"github.com/google/syzkaller/pkg/log"
```

Comment:
The common convention in Go and in syzkaller is to have:
```
import (
  ... standard packages ...
  [empty line]
  ... custom packages ...
)
```

Please do this here and in other files.

Reply:
Sounds good. I don't need to add spaces between packages in trace2syz and other syzkaller packages correct?